packages/blocks/src/api/constants.js

```diff
@@ -10,3 +10,24 @@ export const DEPRECATED_ENTRY_KEYS = [
 	'migrate',
 	'isEligible',
 ];
+
+export const LINK_COLOR = '--wp--style--color--link';
+
+export const STYLE_PROPERTY = {
```

Comment:
I think this is a good place for this kind of info, as it speaks about how the block attributes are structured. However, I don't feel super strongly about this. If this is a blocker I'm fine moving it to any other place.